package.json

```diff
@@ -17,6 +17,6 @@
   },
   "scripts": {
     "test": "echo 'fuck off' && exit",
```

Comment:
are we sure we want to write bad sounding words :P

Reply:
Ok ok... will make it nicer